scripts/zone-generate.sh

```diff
@@ -84,14 +77,14 @@ rm -rf $output $signatures $md5sum && mkdir -p $output
 
 input="$source_dir/africa $source_dir/antarctica $source_dir/asia $source_dir/australasia $source_dir/europe $source_dir/northamerica $source_dir/southamerica"
 input="$input $iso8601"
-input="$input $source_dir/etcetera $source_dir/factory $source_dir/systemv $source_dir/backward"
+input="$input $source_dir/etcetera $source_dir/factory $source_dir/backward"
 #input="$input $source_dir/solar87 $source_dir/solar88 $source_dir/solar89"
 
 for i in $input ; do
   echo "Processing '$i'"
-  $zic -d $output -L /dev/null -y $yearistype $i 2> $build_dir/stderr
-  $zic -d $output/posix -L /dev/null -y $yearistype $i 2>> $build_dir/stderr
-  $zic -d $output/right -L $leapseconds -y $yearistype $i 2>> $build_dir/stderr
+  $zic -d $output -L /dev/null $i #2> $build_dir/stderr
+  $zic -d $output/posix -L /dev/null $i # 2>> $build_dir/stderr
+  $zic -d $output/right -L $leapseconds $i #2>> $build_dir/stderr
   cat $build_dir/stderr | grep -v "time zone abbreviation differs from POSIX standard" >&2 || true
```

Comment:
If you don't direct the output to the file, there is probably no point keeping this line.

Reply:
Actually that redirection removal i was only supposed to do for testing. I'll get it back.